internal/resolution/variablesources/bundles_and_dependencies.go

```diff
@@ -87,7 +87,7 @@ func (b *BundlesAndDepsVariableSource) getEntityDependencies(ctx context.Context
 	// todo(perdasilva): disambiguate between not found and actual errors
 	requiredPackages, _ := bundleEntity.RequiredPackages()
 	for _, requiredPackage := range requiredPackages {
-		semverRange, err := bsemver.ParseRange(requiredPackage.VersionRange)
+		semverRange, err := mmsemver.NewConstraint(requiredPackage.VersionRange)
```

Comment:
Technically, this should remain `blang` if we don't want to change the meaning of the `olm.package.required` constraints.

Reply:
I can look at it; changes were only made if they propagated from `validation.go`s changes. This likely means that it would require intermixing **Masterminds** and **blang** code here.

Reply:
Yeah, looks like you'll need to convert a blang version to a masterminds verison somewhere along the line.

It looks like bundles_and_dependencies.go and required_package.go _both_ call `predicates.InVersionRange`, so it seems like we'll have to break that in two. Perhaps:
- `predicates.InMastermindsVersionRange`
- `predicates.InBlangVersionRange`

And then `predicates.InMasterMindsVersionRange` converts the bundle entity blang version to a masterminds version to check against the mastermind constraint?

Reply:
`blang.Range` is a function and cannot be converted into a `MasterMind.Constraints` (which is a struct). I'm woking out a couple different methods; none of which are pretty...